src/components/Accounts/ChargeFeeFine/ChargeForm.js

```diff
@@ -199,16 +208,24 @@ class ChargeForm extends React.Component {
         </Button>
         <Button
           id="chargeAndPay"
-          disabled={this.props.pristine || this.props.submitting || this.props.invalid}
-          onClick={this.props.handleSubmit(data => this.props.onSubmit({ ...data, pay: true, notify }))}
+          type="submit"
```

Comment:
Why do we need submit here if we have onClick handler? Does this handler will be called when user click enter button?

Reply:
Looks like `type=submit` does not need here.
Yes handler will be called if click enter, if focus button with tab first.